EasyPost.Tests/BetaFeaturesTests/ServicesTests/ReportServiceTest.cs

```diff
@@ -85,11 +88,12 @@ public async Task TestCreateWithColumns()
                 { "columns", columns },
                 { "start_date", Fixtures.ReportDate },
                 { "end_date", Fixtures.ReportDate },
+                { "type", "shipment" },
```

Comment:
question: why make this change? Does it better align with the other libs?

Reply:
Helps make the Parameters set consistent. This was the only function that takes an explicit parameter + a Parameters object.